### ISSUE TITLE
ASE papers are not counted

### ISSUE THREAD

**Issue by intersimone999** [emeryberger/CSrankings#6618]
In the Software Engineering category, ASE papers are currently not counted on the web page, even if the data about such papers are included in `generated-author-info.csv`. Technically, this happens because ASE is in the list of "next tier" conferences (see `csrankings.ts`, line 340).

As far as I know, this matter has been partially discussed in #40. I could not find any more discussion on this (sorry if I missed something). 

Probably, ASE should be marked as a top-tier SE conference: According to CORE Rankings, [it is ranked as A*](http://portal.core.edu.au/conf-ranks/?search=automated+software+engineering&by=all&source=CORE2023&sort=atitle&page=1) (both in CORE2021 and CORE2023) like ICSE and ESEC/FSE.

**Comment by kpmoran** [comment]
I second this comment given that ICSE/FSE/ASE are now all coordinating with one another and are generally considered to be on similar levels in terms of rigor at this point.

**Comment by Kayabar-cyber** [comment]
I fully agree with this suggestion. This is an old issue, would there be any changes by now? @emeryberger

**Comment by darkrsw** [comment]
:+1:

**Comment by luca-digrazia** [comment]
Hi, I follow-up on this and I suggest to have ASE as default venue given its rigor, quality, and accept rate. As a result, SE would have three default venues as ML and security. I think it is fair.

**Comment by darkrsw** [comment]
Fully agreed.
